app/classes/car.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+from app.classes.from_dict import FromDict
+
+
+class Car(FromDict):
+    def __init__(self,
+                 brand: str = "",
+                 fuel_consumption: int | float = 0.0) -> None:
```

Comment:
Do you really want to accept creating cars with "" brand and 0.0 fuel_consumption?
Skip adding default values here, fix everywhere